python/mxnet/kvstore.py

```diff
@@ -11,30 +11,28 @@
 from . import optimizer as opt
 
 def _ctype_key_value(keys, vals):
-    """
-    Returns ctype arrays for the key-value args. For internal use.
-    """
+    names = []
     if isinstance(keys, int):
```

Comment:
testing isinstance is not necessary here. str(.) of string is still string. Always convert when appending